flask_project/campaign_manager/forms/campaign.py

```diff
@@ -49,7 +49,8 @@ class CampaignForm(FlaskForm):
     description = TextAreaField(
         u'Campaign description',
         description='Description for the campaign',
-        render_kw={'placeholder': 'Campaign descriptions'}
+        render_kw={'placeholder': 'Describe the type of features that '
```

Comment:
hi @smit1678 
maybe do it on one line or add +?
'Describe the type of features that ' + 'are being collected and any other participation details.'

i think it breaks python code

Reply:
@meomancer Good catch. I had added two lines because the python linter was raising errors. Added the `+` and it seems to pass the linting still and works correctly. 👍